ts/store/reducers/wallet/payment.ts

```diff
@@ -260,8 +269,8 @@ const isInAllowedOrigins = (
 ): boolean =>
   allowed.some(
     a =>
-      (a === "none" && state.stack.length === 0) ||
-      (state.stack.length > 0 && state.stack[0].kind === a)
+      (state.stack !== null && a === "none") ||
```

Comment:
Shouldn't this be `state.stack === null` ?

Reply:
yes indeed!